src/Request.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs')
+
+class Request {
+  constructor (event) {
+    this.request = this.createRequest(event)
+  }
+
+  getRequest () {
+    return this.request
+  }
+
+  createRequest (event) {
+    const params = event.queryStringParameters
+    const template = event.path === '/journal' ? readTemplate('query_journal_template.json') : readTemplate('query_publisher_template.json')
+    let filterValue = '%'
```

Comment:
reafactor the '%' to a constant with a name describing its meaning

Reply:
Renamed to what it is , an SQL wilcard character